ratis-client/src/main/java/org/apache/ratis/client/RaftClient.java

```diff
@@ -68,6 +68,8 @@
   /** Send set configuration request to the raft service. */
   RaftClientReply setConfiguration(RaftPeer[] serversInNewConf) throws IOException;
 
+  RaftClientRequest createDataStreamHeaderRequest(Message message);
+
```

Comment:
DataStream header is for internal use only in Ratis.  Please don't add this.  (Similar to AsyncApi, the RaftClientRequest is for internal use only.)